alacritty/src/config/bindings.rs

```diff
@@ -111,6 +111,10 @@ pub enum Action {
     /// Store current selection into clipboard.
     Copy,
 
+    #[cfg(not(any(target_os = "macos", windows)))]
+    /// Store current selection into selection buffer.
+    CopySelection,
+
     /// Paste contents of selection buffer.
     PasteSelection,
```

Comment:
Why is `CopySelection` not on macos or windows, while `PasteSelection` is?

Reply:
Historical reasons. `PasteSelection` has no reason to exist on macOS or Windows.